articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,54 @@
+
+## The qubit
+
+In quantum computing, a quantum bit is a unit of quantum information, like a classical bit. Where classical bits hold a single binary value such as a 0 or 1, a qubit can hold both values at the same time. This is called **superposition**.
+
+The act of **measurement** changes a qubit. It goes from being in the infinite states of superposition to one of the binary states of 0 or 1. Again, measurement of a qubit is unlike a classical bit, which when 0 is measured as 0, and when 1 is measured as 1.
+
+Qubits can also be **entangled**: when we make a measurement of one entangled qubit, it affects the state of the other, even if the qubits are physically far apart.
+
+## What can a quantum computer do?
+
+When multiple qubits act coherently, they can process multiple options simultaneously. This allows them to process information in a fraction of the time it would take even the fastest nonquantum systems.
+
+Quantum algorithms can be designed to take advantage of qubit superposition and entanglement to speed up classical algorithms, or to provide entirely new ways of modeling physical systems.
+
+One of the most famous quantum algorithms is `Shor's algorithm` for factorization, which makes the classically intractable problem of factorization of a large number into two prime numbers fast enough to challenge traditional cryptography.
+
+On the more constructive side, algorithms for secure cryptographic key distribution are made possible by superposition, quantum entanglement, and the **no cloning** property of qubits: the inability for qubits to be copied without detection.
+
+`Grover's algorithm` provides a quadratic factor speed-up for database searching.
+
+Using quantum programs to model quantum systems themselves is another application of quantum computing. Photosynthesis, superconductors, and chemical molecules are examples of quantum systems that can be simulated using quantum programs.
+
+## Quantum hardware
+
+In classical computers, bits correspond to voltage levels in silicon circuits. Quantum computing hardware can be implemented by many different physical realizations of qubits: trapped ions, superconducting, neutral atoms, electron spin, light polarization, topological qubits. Quantum hardware is an emergent technology. It requires balancing fidelity of the system with scalability. The larger the scale (that is, number of qubits), the higher the error rate.
+
+## Quantum simulations
+
+Up to a certain scale (number of qubits), quantum computing can be simulated on a classical computer. Using simulations is how you can start to write quantum programs using Microsoft Quantum, the Quantum Development Kit (QDK), and the Q# language.
+
+## Get started today
+
+* Build a quantum program that implements [Grover's algorithm](xref:microsoft.quantum.quickstarts.search)
+* Take a quick look at Q# by developing a [Bell State](xref:microsoft.quantum.write-program) program
```

Comment:
Perhaps toss the hyperlinks on the whole sentences or just the first phrase? With it on just that the key word it sort of seems like it'll be a hyperlink to the definition rather than a quickstart